ci/vm/entrypoint.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash -xe
+
+OS=$1
+
+install_packages() {
+  if [[ $OS == *"ubuntu"* ]]; then
+    apt-get update &&
+      DEBIAN_FRONTEND=noninteractive apt-get install -yq \
+        automake \
+        dkms \
+        git \
+        libtool &&
+      apt-get clean && rm -rf /var/lib/apt/lists/*
+  elif [[ $OS == *"centos"* ]]; then
+    yum install -y -q centos-release-scl
+    yum install -y -q \
+      automake \
+      devtoolset-8-gcc \
+      devtoolset-8-gcc-c++ \
+      elfutils-libelf-devel \
+      git \
+      libtool \
+      make &&
+      yum clean all
+  fi
+}
+
+xpmem_build() {
+  git clone -q https://github.com/openucx/xpmem.git
+  cd xpmem
+  ./autogen.sh
+  if [[ $OS == *"ubuntu"* ]]; then
+    ./configure --with-kerneldir=/usr/src/linux-headers-"$(uname -r)"
+    make -s
+    make check
+  elif [[ $OS == *"centos"* ]]; then
+    # Build with GCC-8
+    scl enable devtoolset-8 -- bash -c "
+      ./configure --with-kerneldir=/usr/src/kernels/'$(uname -r)'
+      make -s
+      make check
+    "
+  fi
+}
+
+xpmem_load() {
+  sudo insmod kernel/xpmem.ko
+  if ! lsmod | grep -q xpmem; then
```

Comment:
i'd suggest to check xpmem srcversion (in /sys/module/xpmem/srcversion) matches the file - from "modinfo kernel/xpmem.ko|grep srcversion"

Reply:
Good idea, implemented.